src/components/10-atoms/button/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 4.1.0
+
+- You can now set css `width` and the `button` always look quite nice. (#1686).
```

Comment:
@danyball: You are able to set the css `width` property and the `button` looks always nice.

Reply:
Don't like the expression "quite nice". It is not measurable and the user doesn't know what really changed. I would rewrite it.

Reply:
Changed it to `nice`. Or do you want  more technical details?

Reply:
Or something like: "You can now set a custom css `width` for the `button` to fit your needs"

Reply:
You could do that before. But now the button looks good with it. I could also write that now the label is centered.

Reply:
Ah okay, then choose what describes it the best in your opinion👍🏼